eng/pipelines/performance/templates/perf-ios-scenarios-build-jobs.yml

```diff
@@ -17,13 +17,6 @@ jobs:
         postBuildSteps:
           - template: /eng/pipelines/performance/templates/build-perf-sample-apps.yml
             parameters:
-              rootFolder: '$(Build.SourcesDirectory)/artifacts/'
-              includeRootFolder: true
-              displayName: iOS Mono Artifacts
-              artifactName: iOSMonoarm64
-              archiveExtension: '.tar.gz'
-              archiveType: tar
-              tarCompression: gz
```

Comment:
Why is this no longer needed?

Reply:
All except the artifact name were not being used at all in the app build yaml, so they were just left over from when I picked up the project. For the artifact name, we were using it with the configuration information to generate the names of the artifacts, but since it was only used for iOS and only a handful of places, I decided to remove it with the other options.